pkg/services/audit/auditor/pdp.go

```diff
@@ -108,7 +108,9 @@ func (c *Context) splitPayload(id *object.ID) []uint64 {
 
 func (c *Context) collectHashes(p *gamePair) {
 	fn := func(n *netmap.Node, rngs []*object.Range, hashWriter func([]byte)) {
-		// TODO: add order randomization
+		rand.Shuffle(len(rngs), func(i, j int) {
+			rngs[i], rngs[j] = rngs[j], rngs[i]
+		})
```

Comment:
It seems to me that this change is not quite correct. @cthulhu-rider, can you double check? I think original TODO meant to randomize order of network calls (`c.cnrCom.GetRangeHash()`). But those calls are strictly tied with `hashWriter()` and it makes the logic incorrect.

Hashes of provided `rngs` are appended into a `hh1` and `hh2` lists. 
```go
	p.hh1 = make([][]byte, 0, len(p.rn1))

	fn(p.n1, p.rn1, func(h []byte) {
		p.hh1 = append(p.hh1, h)
	})

	p.hh2 = make([][]byte, 0, len(p.rn2))

	fn(p.n2, p.rn2, func(h []byte) {
		p.hh2 = append(p.hh2, h)
	})
```

The order of hashes in this list is important, because they are matched to each other after homomorphic concatenation. 

```
|         hh1[0]         |   hh1[1]  |   hh1[2]  |
|------------|-----------|-----------|-----------| : object ranges
|   hh2[0]   |   hh2[1]  |         hh2[2]        |
```

Reply:
Yeah, removed this commmit.